packages/components/search/src/search.js

```diff
@@ -138,7 +138,7 @@ export class TSSearch extends TSElement {
 					return;
 				}
 				/**
-				 * Emitted on every user's change in a search input
+				 * Emitted on every user's change in a search input or when user selects from a dropdown
```

Comment:
I think it would be more clear if we say this instead: 
```
				 * Emitted on every user's change in a search input or when user selects an item from the provided `dropdownItems`
```
What do you think?

Reply:
Sounds good to me. I will update it.